### PR TITLE
add parameter 'host' in cinder.conf

### PR DESCRIPTION
**Pull request by douyq**
host = cinder is needed in cinder.conf

### CODE REVIEW

**Comment by nirajdp76** [comment]
More background information to help explain the need for the the fix
 
Currently in our deployment model we have 2 cinder-volume services running; one on each controller. By default when volume is created, it will assign the os-vol-host-attr:host to host on which volume service handled the create request. Since we are using ceph as backend we should have a unique host value (say ceph). This will allow either of volume service to operate on the volume (delete/attach/detach)

Note: For existing production environments, we have to ensure cinder volume os-vol-host-attr:host attribute on existing volumes are updated. This should not have any impacts to data path.

File: roles/cinder-common/templates/etc/cinder/cinder.conf [hunk 1]
```diff
@@ -28,6 +28,8 @@ auth_strategy = keystone
 volume_clear_size = {{ cinder.volume_clear_size }}
 {% endif -%}
 
+host = cinder
```
Comment (nirajdp76) [inline roles/cinder-common/templates/etc/cinder/cinder.conf]: We should set this variable only when ceph is enabled.

**Comment by j2sol** [comment]
ok to test

**Comment by j2sol** [comment]
Ceph is disabled in CI. Going to merge this, shouldn't impact CI.

**Pull request merged**